test/unit-test/test_lib/test_analysis/test_network.py

```diff
@@ -306,25 +306,27 @@ def test_nick_receiver_from_conn_comp(self):
     @mock.patch('lib.analysis.network.message_number_graph', autospec=True)
     @mock.patch('lib.analysis.network.user.keywords', autospec=True)
     def test_identify_hubs_and_experts(self, mock_keywords, mock_msg_graph):
-    
-        top_hub_ = util.load_from_disk(current_directory+ "/data/top_hub")
-        top_keyword_overlap_ = util.load_from_disk(current_directory+ "/data/top_keyword_overlap")
-        top_auth_ = util.load_from_disk(current_directory+ "/data/top_auth")
-        message_graph = util.load_from_disk(current_directory+ "/data/test_hits_message_graph")
-        keyword_dict_list = util.load_from_disk(current_directory+ "/data/keyword_dict_list")
-        user_keyword_freq_dict = util.load_from_disk(current_directory+"/data/user_keyword_freq_dict")
-        user_words_dict_list = util.load_from_disk(current_directory+"/data/user_words_dict_list")
-        nicks_for_stop_words = util.load_from_disk(current_directory+"/data/nicks_for_stop_words")
-        keywords_for_channels = util.load_from_disk(current_directory+"/data/keywords_for_channels")
-        keywords_return = util.load_from_disk(current_directory+ "/data/keywords_return")
+        
+        log_data = util.load_from_disk(current_directory+ "/data/hits/log_data")
+        nicks = util.load_from_disk(current_directory+ "/data/hits/nicks")
+        nick_same_list = util.load_from_disk(current_directory+ "/data/hits/nick_same_list")
+        top_hub_ = util.load_from_disk(current_directory+ "/data/hits/top_hub")
+        top_keyword_overlap_ = util.load_from_disk(current_directory+ "/data/hits/top_keyword_overlap")
+        top_auth_ = util.load_from_disk(current_directory+ "/data/hits/top_auth")
+        message_graph = util.load_from_disk(current_directory+ "/data/hits/message_graph")
+        keyword_dict_list = util.load_from_disk(current_directory+ "/data/hits/keyword_dict_list")
+        user_keyword_freq_dict = util.load_from_disk(current_directory+"/data/hits/user_keyword_freq_dict")
+        user_words_dict_list = util.load_from_disk(current_directory+"/data/hits/user_words_dict_list")
+        nicks_for_stop_words = util.load_from_disk(current_directory+"/data/hits/nicks_for_stop_words")
+        keywords_for_channels = util.load_from_disk(current_directory+"/data/hits/keywords_for_channels")
         
         # setup mock
         mock_msg_graph.return_value = message_graph
         mock_keywords.return_value = keyword_dict_list, user_keyword_freq_dict, user_words_dict_list, nicks_for_stop_words, keywords_for_channels
         capturedOutput = StringIO.StringIO()
         sys.stdout = capturedOutput
         
-        message_num_graph, top_hub, top_keyword_overlap, top_auth = network.identify_hubs_and_experts(self.log_data, self.nicks, self.nick_same_list)
+        message_num_graph, top_hub, top_keyword_overlap, top_auth = network.identify_hubs_and_experts(log_data, nicks, nick_same_list)
```

Comment:
If the expanded logs can be used for other methods of the network.py module, we should use them instead of keeping one copy of logs per unit test.

Reply:
@prasadtalasila WIth the common logs, all the lines were not covered and hence I had to generate a slightly big log data.

Reply:
Which means these logs would also be useful for other unit test functions of the network.py module. Can we reuse the new logs in those tests and remove their existing logs?

Reply:
@prasadtalasila Sir, most of the lines (except 3) are covered so I don't think we will gain much. Also, this log data is for 4 days, for functions like message_number/time_graphs we are storing big lists and the sizes can bloat up very easily with even slight increase log duration due the unnecessary repetitive work being done. These files easily take few Mbs of disk space even for log data of 1/2 days.